docs/Reference/CLI/CLI-Syntax.md

```diff
@@ -65,6 +66,19 @@ Endpoint to which received requests are forwarded. Default is `localhost`.
 --downstream-http-host=192.168.05.14
 ```
 
+### `downstream-http-path`
+
+Path to which received requests are forwarded. Default is `/`. Required if [connecting to a
```

Comment:
This might not be required for all cloud-based Ethereum clients, but it certainly is for Infura. Perhaps just say "Might be required if ...."

Reply:
Done